packages/react-components/react-textarea/src/stories/TextareaAppearance.stories.tsx

```diff
@@ -1,18 +1,27 @@
 import * as React from 'react';
-import { useId } from '@fluentui/react-utilities';
-import { Textarea } from '../index';
 import { Label } from '@fluentui/react-label';
 import { makeStyles, shorthands } from '@griffel/react';
+import { Text } from '@fluentui/react-text';
+import { Textarea } from '../index';
+import { useId } from '@fluentui/react-utilities';
 
 const useStyles = makeStyles({
-  container: {
+  base: {
     display: 'flex',
     flexDirection: 'column',
     ...shorthands.gap('10px'),
-    '& > div': { display: 'flex', flexDirection: 'column', ...shorthands.gap('10px'), ...shorthands.padding('10px') },
+    '& > div > div': {
+      display: 'flex',
+      flexDirection: 'column',
+      ...shorthands.gap('10px'),
```

Comment:
This can be handled in a separate PR but we should remove flex-gap usage from our stories.

Reply:
I'll make a follow-up PR removing gap from all the stories.